addons/crm/static/tests/tours/crm_rainbowman.js

```diff
@@ -14,20 +14,37 @@
             trigger: ".o-kanban-button-new",
             content: "click create",
         }, {
-            trigger: "input[name=name]",
+            trigger: ".o_field_widget[name=name] input",
             content: "complete name",
             run: "text Test Lead 1",
         }, {
-            trigger: "div[name=expected_revenue] > input",
+            trigger: ".o_field_widget[name=expected_revenue] input",
             content: "complete expected revenue",
             run: "text 999999997",
         }, {
             trigger: "button.o_kanban_add",
             content: "create lead",
+        }, { //OWL BUG: can not move a newly created record directly, go to lead's form view and back.
+            trigger: ".o_kanban_record .o_kanban_record_title:contains('Test Lead 1')",
+            content: "open lead",
+        }, {
+            trigger: '.o_back_button',
+            content: 'navigate back to the kanban view',
+            position: "bottom",
+            run: "click",
```

Comment:
It's as you prefer, I was just making the branch green, I already reported the bug and hopefully it can be fixed soon :)
I was told this PR was blocking some other stuff.